pkg/progressbar/progress_test.go

```
@@ -53,16 +53,25 @@
+			req, err := os.CreateTemp("", "test_req")
+			assert.NoError(t, err)
+
+			_, err = algo.WriteString("test algorithm")
+			assert.NoError(t, err)
+
+			_, err = req.WriteString("test request")
+			assert.NoError(t, err)
 
 			algoStream := new(mocks.AgentService_AlgoClient)
-			algoStream.On("Send", mock.Anything).Return(tc.sendError)
+			//algoStream.On("Send", mock.Anything).Return(tc.sendError)
```

golangci-lint

File is not `gofumpt`-ed

(gofumpt)